lib/radiator_web/controllers/api/audio_file_controller.ex

```diff
@@ -0,0 +1,35 @@
+defmodule RadiatorWeb.Api.AudioFileController do
+  use RadiatorWeb, :controller
+
+  alias Radiator.Media
+
+  action_fallback(RadiatorWeb.Api.FallbackController)
+
+  def create(conn, %{"audio_file" => audio_file_params}) do
+    case Media.create_audio_file(audio_file_params) do
```

Comment:
since we use the fallback-controller i would suggest using `with` instead of case and let all the errors just fall through and handle upward.